.github/workflows/integration-test-deployment.yml

```diff
@@ -100,8 +100,9 @@ jobs:
       - name: Build deployment-integ
         run: yarn --cwd tools/@aws-cdk/integration-test-deployment build
         
-      - name: Build Integration Tests
-        run: npx lerna run build --scope=@aws-cdk-testing/framework-integ 
+      # Build entire CDK for integration test packages: @aws-cdk-testing/framework-integ (stable tests) and @aws-cdk (alpha tests)
+      - name: Build Integration Test packages 
+        run: npx lerna run build --skip-nx-cache
```

Comment:
No need to skip the nx cache, you can remove that.

Reply:
You can also selectively build alpha packages with the option `--scope="@aws-cdk/*"`